site/_docs/reference.md

```diff
@@ -2813,6 +2813,7 @@ In the following:
 | b | PARSE_TIMESTAMP(format, string[, timeZone])    | Uses format specified by *format* to convert *string* representation of timestamp to a TIMESTAMP WITH LOCAL TIME ZONE value in *timeZone*
 | h s | PARSE_URL(urlString, partToExtract [, keyToExtract] ) | Returns the specified *partToExtract* from the *urlString*. Valid values for *partToExtract* include HOST, PATH, QUERY, REF, PROTOCOL, AUTHORITY, FILE, and USERINFO. *keyToExtract* specifies which query to extract
 | b s | POW(numeric1, numeric2)                      | Returns *numeric1* raised to the power *numeric2*
+| p | RANDOM()                                       | Generates a random double between 0 and 1 inclusive
```

Comment:
Is there already documentation for RAND? If so, just say `Equivalent to RAND` (see other functions w multiple names for examples)

Reply:
This is from the RAND documentation, minus the comment about an optional seed. In this case where the functions are not exactly the same due to the missing overload, should we still use "Equivalent to"...?

Reply:
ah sorry didn't see there's a difference, this is fine